test/test_cli_integration.py

```diff
@@ -136,28 +142,32 @@ def generate_policy_string(self):
         variable_3 = 'simple/special @#$%^&*(){{}}[]._+/{id}'.format(id=uuid.uuid4().hex)
 
         policy = \
-"""
-- !variable
-  id: {variable_1}
-- !variable
-  id: {variable_2}
-- !variable
-  id: {variable_3}
-"""
+            """
+            - !variable
+              id: {variable_1}
+            - !variable
+              id: {variable_2}
+            - !variable
+              id: {variable_3}
+            """
 
         dynamic_policy = policy.format(variable_1=variable_1,
                                        variable_2=variable_2,
                                        variable_3=variable_3)
 
         return (dynamic_policy, [variable_1, variable_2, variable_3])
 
-
     # *************** TESTS ***************
-
     @integration_test
     def test_http_cli_can_set_and_get_a_defined_variable(self):
         self.setup_cli_params(self.HTTP_ENV_VARS)
+        self.set_variable(CliIntegrationTest.DEFINED_VARIABLE2, "r")  # "!@#$%asdfgh&*()g,lpokmnjiub")
+        self.set_variable(CliIntegrationTest.DEFINED_VARIABLE3, "1")
```

Comment:
please giving meaningful names and values
`DEFINED_VARIABLE_TWO` or `DEFINED_VARIABLE_ILLEGAL_CHARS` 
`DEFINED_VARIABLE_THREE somevalue`